website/blog/2021-06-18-first-GA-version-v1.0-of-Apache-APISIX-Ingress-Controller-released.md

```diff
@@ -0,0 +1,121 @@
+---
+title: "Apache APISIX Ingress Controller 首个 GA 版本 v1.0 正式发布！"
+author: Jintao Zhang
+authorURL: "https://github.com/tao12345666333"
+authorImageURL: "https://avatars.githubusercontent.com/u/3264292?v=4"
+---
+> [@tao12345666333](https://github.com/tao12345666333), Apache APISIX commiter from [Shenzhen Zhiliu Technology Co.](https://www.apiseven.com/)
+>
+
+<!--truncate-->
+### 关于 Apache APISIX Ingress Controller
+
+Apache APISIX Ingress Controller 是一个使用 Apache APISIX 作为数据面承载流量的云原生 Ingress Controller 实现，采用 CRD 的方式对 Kubernetes 进行了扩展。
+
+可支持使用包括 ApisixRoute、ApisixUpstream 等自定义资源，以及 Kubernetes 原生 Ingress 资源等控制外部流量访问部署在 Kubernetes 中的服务。
+
+整体架构如下：
+
+![图片](../static/img/blog_img/2021-06-18-1.png)
+
+### v1.0 最新特性
+
+#### 增加 ApisixConsumer 自定义资源使配置认证更方便
+
+在之前的版本中，如果想要配置 keyAuth 或者 basicAuth 需要手动去调用 Apache APISIX 的 admin api 创建 consumer 配置。
+
+在 v1.0 版本中，我们增加了 `ApisixConsumer` 资源，用户可以使用更原生的方式来定义 consumer 资源及为 ApisixRoute 配置认证。
+
+例如通过以下配置定义一个 keyAuth 资源：
+
+```text
+apiVersion: apisix.apache.org/v2alpha1
+
+kind: ApisixConsumer
+
+metadata:
+
+  name: keyauth
+
+spec:
+
+  authParameter:
+
+    keyAuth:
+
+      value:
+
+        key: API
+```
+
+在 ApisixRoute 中只需要增加对应类型的 `authentication` 配置即可。
+
+```text
+apiVersion: apisix.apache.org/v2alpha1
+
+kind: ApisixRoute
+
+metadata:
+
+ name: httpbin-route
+
+spec:
+
+ http:
+
+   ...
+
+   authentication:
+
+     enable: true
+
+     type: keyAuth
+```
+
+#### 为 ApisixTls 增加 mTLS 支持
+
+在 v1.0 版本中我们也为 ApisixTls 自定义资源增加了 mTLS 的支持，只需要在 ApisixTls 资源的配置中增加 client 配置即可，例如：
+
+```text
```

Comment:
Too many empty lines, should remove them so that the format can be compact.

Reply:
let me help @Serendipity96 to update this PR